src/endpoints/http/collections/delete.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+
+function message(message) {
+  return { message };
+}
+/**
+ * Add the route to delete collection(s) for the users App.
+ * This route expects a db connection to be present on the request object.
+ * 
+ * @param  {Object} router - The express router.
+ */
+export function deleteCollections(router) {
+  router.post('/collections', (req, res, next) => {
+    const reqCollections = req.body.names;
+    if (!reqCollections) {
+      return res.status(400).send(message('names(s) of collection(s) is required'));
+    }
+    const allCollectionNames = req.db.getCollectionNames();
+    if (!allCollectionNames) {
+      return res.status(400).send(message('No collections exists in database'));
+    }
+    _.filter(reqCollections, function(name) {
```

Comment:
The way I'm checking if the collection names are valid is through the `includes()` function. The `filter()` function iterates over the array `reqCollections` and if an element exists in the `allCollectionNames` array then I get the collection by `req.db.getCollection(name)`. I'll add in that an error is returned if it doesn't exist